src/reducers/passwordReminder/passwordReminderReducer.test.js

```diff
@@ -0,0 +1,284 @@
+/* globals test describe expect */
```

Comment:
In the core I've had to start doing `import { describe, it } from 'mocha'`, which was the only way to avoid Flow errors. I'm surprised this works, but maybe Jest is using some hack to get these in Flow's global namespace.